tests/unit_tests/test_wave_02.py

```diff
@@ -34,7 +34,6 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("electronics")
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    #raise Exception("Complete this test according to comments below.")
+    assert len(items) == 0
+    assert items == []
```

Comment:
I would say both of these asserts check the same thing (whether `items` is empty). Since we want to know that it is a list and it is empty, we could just keep the second check `assert items == []`. 

Aside from confirming the list is empty, what could we assert to make sure the `vendor`'s inventory was not changed?